spec/helpers/application_helper_spec.rb

```diff
@@ -60,6 +60,24 @@
     end
   end
 
+  describe 'date_string' do
+    it 'If the conference is only one day long' do
```

Comment:
A better wording for your tests would be
```
context 'returns correct string'
  it 'when conference lasts 1 day'
  it 'when conference starts and ends in the same month and year'
  it 'when conference ends in another month, of the same year'
  it 'when conference ends in another month, of a different year'
```

Generally it is common practice to use **when** in it blocks, and if the test was to fail, it would read:

> #date_string returns correct string when conference lasts 1 day